app/src/main/java/com/snatik/matches/fragments/DifficultySelectFragment.java

```diff
@@ -50,7 +56,43 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 
 		animate(difficulty1, difficulty2, difficulty3, difficulty4, difficulty5, difficulty6);
 
+		Typeface type = Typeface.createFromAsset(Shared.context.getAssets(),"fonts/grobold.ttf");
+
+
+        TextView text1 = (TextView) view.findViewById(R.id.time_difficulty_1);
+		text1.setGravity(Gravity.CENTER_HORIZONTAL | Gravity.CENTER_VERTICAL);
+		text1.setTypeface(type);
+        text1.setText(Memory.getBestTimeForStage(theme.id,1));
+
+        TextView text2 = (TextView) view.findViewById(R.id.time_difficulty_2);
+		text2.setGravity(Gravity.CENTER_HORIZONTAL | Gravity.CENTER_VERTICAL);
+		text2.setTypeface(type);
+        text2.setText(Memory.getBestTimeForStage(theme.id,2));
+
+        TextView text3 = (TextView) view.findViewById(R.id.time_difficulty_3);
+		text3.setGravity(Gravity.CENTER_HORIZONTAL | Gravity.CENTER_VERTICAL);
+		text3.setTypeface(type);
+        text3.setText(Memory.getBestTimeForStage(theme.id,3));
```

Comment:
code formating. please do it in this class.

Reply:
This is how the file looks on my computer. Don't know why it changes when I push it. 
![image](https://user-images.githubusercontent.com/15691073/30431680-0c2863d2-9968-11e7-8129-004c6c2ee7f1.png)